queries/gdshader/highlights.scm

```diff
@@ -0,0 +1,142 @@
+[
+  "render_mode"
+  "shader_type"
+  "group_uniforms"
+  "global"
+  "instance"
+  "const"
+  "varying"
+  "uniform"
+  "struct"
+  "case"
+  "default"
```

Comment:
These should also be `@keyword.conditional`.

Reply:
done